docs/releases/v2018.2.4.rst

```diff
@@ -0,0 +1,52 @@
+Gluon 2018.2.4
+==============
+
+Important Note
+~~~~~~~~~~~~~~
+
+This release will be the last one of the Gluon branch v2018.2.x , an update to v2019.1.x is strongly recommended and should be fairly easy.
+
+Bugfixes
+~~~~~~~~
+
+* Fixes device alias for Ubiquiti UniFi AC LR. (`#1834 <https://github.com/freifunk-gluon/gluon/issues/1834>`_)
```

Comment:
```suggestion
* Fixes device alias for Ubiquiti UniFi AC LR. (`#1834 <https://github.com/freifunk-gluon/gluon/issues/1834>`_)
  Autoupdates on this model were impossible before, since we were missing the proper device alias.
```